build/maven/xmlcodegen/pom.xml

```diff
@@ -57,16 +57,4 @@
 
   </dependencies>
 
-  <build>
```

Comment:
You're right, I've missed this one. Eliminating the block does not match the same behavior though, this module should probably not be formatted (so we need to find a way to skip it).

Reply:
There was a reason indeed it seems, reformatting causes:

``
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-plugin-plugin:3.6.0:descriptor (default-descriptor) on project xmlcodegen: Error extracting plugin descriptor: 'The description element in parameter # 0 is invalid. It cannot be null.' -> [Help 1]
``

Oh well, keeping the skipping then :-D